test/Confluent.Kafka.IntegrationTests/Tests/AssignPastEnd.cs

```diff
@@ -46,14 +46,19 @@ public static void AssignPastEnd(string bootstrapServers, string topic)
             using (var producer = new Producer<Null, string>(producerConfig, null, new StringSerializer(Encoding.UTF8)))
             {
                 dr = producer.ProduceAsync(topic, null, testString).Result;
+                Assert.NotEqual(dr.Offset, Offset.Invalid);
```

Comment:
`>= 0` might be better here since it excludes a larger set of values we don't want to see